packages/synapse-react-client/src/theme/palette/Palettes.ts

```diff
@@ -88,6 +88,8 @@ export const palette: PaletteOptions = {
     200: '#f1f3f5',
     100: '#fbfbfc',
   }),
+  darkPrimary: generatePalette('#0b1218'), // primary-900
+  light: { ...generatePalette('#f8f9fa'), contrastText: '#22252a' }, // grey-1000
```

Comment:
Override generatePalette's default contrastText value (white) so that light button text is visible